PythonRegression/tests/features/machine1/1_api_tests.feature

```diff
@@ -38,56 +53,174 @@ Feature: Test API calls on Machine 1
+        |keys       |values                 |type               |
+        |addresses  |TEST_EMPTY_ADDRESS     |staticList         |
+        |threshold  |100                    |int                |
+
+
+        Then a response with the following is returned:
+        |keys                               |
+        |balances                           |
+        |duration                           |
+        |milestoneIndex                     |
+        |references                         |
+
+
+    Scenario: Interrupt attach to tangle
+        Given "attachToTangle" is called on one thread in "nodeA" with:
+        |keys                   |values                         |type           |
+        |trytes                 |EMPTY_TRANSACTION_TRYTES       |staticList     |
+        |trunk_transaction      |TEST_HASH                      |staticValue    |
+        |branch_transaction     |TEST_HASH                      |staticValue    |
+        |min_weight_magnitude   |50                             |int            |
+
+        And "interruptAttachingToTangle" is called on one thread in "nodeA" with:
+        |keys                   |values                         |type           |
+
+        When all api call threads are joined
+        #The only active thread should be the main testing thread
+        Then while checking for running threads, there should be "1" running
+
+
+
+
+
+
+    Scenario: WereAddressesSpentFrom is called
+        Given "wereAddressesSpentFrom" is called on "nodeA" with:
+        |keys       |values                 |type               |
+        |addresses  |TEST_EMPTY_ADDRESS     |staticList         |
+
+    	Then a response with the following is returned:
+    	|keys                               |
+    	|duration                           |
+    	|states                             |
+
+
+
+    Scenario: Create, attach, store and find a transaction
+        Generate a transaction, attach it to the tangle, and store it locally. Then find
+        that transaction via its address.
+
+        Given a transaction is generated and attached on "nodeA" with:
+        |keys       |values                 |type           |
+        |address    |TEST_STORE_ADDRESS     |staticValue    |
+        |value      |0                      |int            |
+
+        Then a response with the following is returned:
+        |keys                               |
+        |trytes                             |
+
+        When "storeTransactions" is called on "nodeA" with:
+        |keys       |values                 |type           |
+        |trytes     |TEST_STORE_TRANSACTION |staticValue    |
+
+        And "findTransactions" is called on "nodeA" with:
+        |keys       |values                 |type           |
+        |addresses  |TEST_STORE_ADDRESS     |staticList     |
+
+        Then a response with the following is returned:
+        |keys                               |
+        |hashes                             |
```

Comment:
Since we have a static db, maybe you can check the returned values and not just the keys